listenbrainz/webserver/templates/navbar.html

```diff
@@ -58,6 +42,22 @@
             <li><a href="{{ url_for('index.similar_users') }}">Top similar users</a></li>
           </ul>
         </li>
+        <li class="dropdown">
```

Comment:
The about menu in the middle of the menu bar felt weird, thus I moved it to the end. Looks much better.